Automattic/wp-calypso
# GM16/Republicize

Introduces functionality to share messages on social media multiple times
Part of GM16 project

##Testing 

- Connect twitter / facebook, etc
- Go to posts list
- Edit published post
- go to sharing
- See "Share" button
- Click it
- See that message goes to your connected social media
- Return to posts lict
- See megaphone at the bottom of any published post
- Click it
- See similar functionality
- Edit mssage
- Click "Share again"
- See that it goes out to your twitter / etc
- Dismiss success message

## Visuals

<img width="742" alt="zrzut ekranu 2016-09-17 o 16 50 02" src="https://cloud.githubusercontent.com/assets/3775068/18611733/88189d10-7cf7-11e6-9ecf-43129ead4f6f.png">
